src/lib/consts/store.ts

```diff
@@ -0,0 +1,6 @@
+type Store = `${string}_STORE`
```

Comment:
IMO, I think typing the key parameter to `storage` gives better type guidance to the next dev. If you do that, you can just create string literal types without the need for declaring constant strings.
```suggestion
type Store = 'CHAT_STORE' | 'THEME_STORE'
```

Then use the `Store` type as the 'key' parameter type, and TS will only allow those two strings as arguments 
```typescript
const storage = <T, K extends Store>(key: K, initValue: T): Writable<T> => { }
// OR
const storage = <T>(key: Store, initValue: T): Writable<T> => { }
```
```typescript
export const themeStore = storage<Theme>('THEME_STORE', getTheme(getOSTheme()));
```

Reply:
Can't we just move this type to `types` and just remove all the `consts`?

Reply:
Ah yeah you did exactly that